### DISCUSSION TITLE
Different views for different folders.

### DISCUSSION THREAD

**Discussion by DigitalRustt** [derceg/explorerplusplus#505] [Ideas]
Can you implement the ability to display a different view on each folder (thumbnail, large icon, small icon etc.) the same way windows file explorer does it? I believe right now the view setting applies globally to everything, so for instance you have to constantly switch between detail view or thumbnail view if you find yourself in a directory full of folders or a directory full of pictures. One way of implementing it would be to remember the view that has been set on each folder, I think windows file explorer does it that way.

**Comment by DejayRezme** [comment]
Most importantly some folders, especially the Downloads folder, should always be sorted by newest date first. And you need to click the column twice to switch. 

BTW using Ctrl+Scrolwheel lets you quickly switch from thumbnail to details. (I'd love larger thumbnails though. And will smaller margins. And a quickview for images haha)
